working/wildcards/feature-specification.md

```diff
@@ -228,7 +239,7 @@ class C {
   var other;
 
   C(this._)
-    : other = _ { // <-- Error. No "_" in scope.
+    : other = _ { // <-- Error, cannot access `this`.
```

Comment:
`_` is actually in scope because it is an instance member, but it is not allowed for an initializing expression in this context to access `this`. Hence the updated comment.